applications/main/bad_usb/scenes/bad_usb_scene_config.c

```diff
@@ -13,12 +65,51 @@ void bad_usb_scene_config_select_callback(void* context, uint32_t index) {
 
 static void draw_menu(BadUsbApp* bad_usb) {
     VariableItemList* var_item_list = bad_usb->var_item_list;
+    VariableItem* item;
 
     variable_item_list_reset(var_item_list);
 
     variable_item_list_add(var_item_list, "Keyboard Layout (global)", 0, NULL, NULL);
 
-    variable_item_list_add(var_item_list, "Remove Pairing", 0, NULL, NULL);
+    item = variable_item_list_add(
+        var_item_list, "Connection", 2, bad_usb_scene_config_connection_callback, bad_usb);
+    variable_item_set_current_value_index(item, bad_usb->interface == BadUsbHidInterfaceBle);
+    variable_item_set_current_value_text(
+        item, bad_usb->interface == BadUsbHidInterfaceBle ? "BLE" : "USB");
+
+    if(bad_usb->interface == BadUsbHidInterfaceBle) {
+        BleProfileHidParams* ble_hid_cfg = &bad_usb->script_hid_cfg.ble;
+
+        item = variable_item_list_add(
+            var_item_list, "BLE Remember", 2, bad_usb_scene_config_ble_remember_callback, bad_usb);
+        variable_item_set_current_value_index(item, ble_hid_cfg->bonding);
+        variable_item_set_current_value_text(item, ble_hid_cfg->bonding ? "ON" : "OFF");
+
+        item = variable_item_list_add(
+            var_item_list,
+            "BLE Pairing",
+            GapPairingCount,
+            bad_usb_scene_config_ble_pairing_callback,
+            bad_usb);
+        variable_item_set_current_value_index(item, ble_hid_cfg->pairing);
+        variable_item_set_current_value_text(item, ble_pairing_names[ble_hid_cfg->pairing]);
+
+        variable_item_list_add(var_item_list, "BLE Device Name", 0, NULL, NULL);
+
+        variable_item_list_add(var_item_list, "BLE MAC Address", 0, NULL, NULL);
+
+        variable_item_list_add(var_item_list, "Randomize BLE MAC", 0, NULL, NULL);
+
+        variable_item_list_add(var_item_list, "Remove BLE Pairing", 0, NULL, NULL);
+    } else {
+        variable_item_list_add(var_item_list, "USB Manufacturer", 0, NULL, NULL);
```

Comment:
Same here. Maybe call items that are actionable according to the action? "Set Manufacturer Name", "Set Product Name", etc